util/reggen/gen_dv.py

```diff
@@ -14,7 +15,6 @@
 from pkg_resources import resource_filename
 
 from .ip_block import IpBlock
-from .multi_register import MultiRegister
```

Comment:
Please revert these two hunks: it's probably clearer to keep the reggen imports together.

Reply:
fixed